Project6.md

```diff
@@ -50,10 +48,12 @@ Please turn in via email a write up including:
 
 2. updated docker-compose.yml if you are running any new or changed containers
  
-2. link to any code in github you wrote for this (especially for caching and unit test work)
+2. link to any code in github you wrote for this (especially for caching)
 
-3. include any sample output (especially for the load balancer, unit test, performance test examples)
+3. include any sample output (especially for the load balancer, continuous integration, performance test examples)
 
 4. include any command lines (such as for running load tests, unit tests or load balancers)
 
 5. for hosting on digital island, include a link to your app or service working there
+
+6. Don't forget to provide user stories and unit tests accordingly.
```

Comment:
Does it make sense to require new user stories and unit tests for this project? It's all infrastructure and not user facing so not sure what we'd expect students to do?